docs/topics/gradle.md

```diff
@@ -968,3 +968,74 @@ When configuring the Kotlin daemon's JVM arguments, note that:
   >
   {type="note"}
 * If the `Xmx` is not specified, the Kotlin daemon will inherit it from the Gradle daemon.
+
+## Defining Kotlin compiler execution strategy
+
+_Kotlin compiler execution strategy_ describes where Kotlin compiler is executed and if incremental compilation is supported in each case.
+
+There are three compiler execution strategies:
+
+| Strategy       | Where Kotlin compiler is executed | Support for incremental compilation | Other characteristics                                                  |
+|----------------|-----------------------------------|-------------------------------------|------------------------------------------------------------------------|
+| Daemon         | Inside its own daemon process     | Yes                                 | *The default strategy*. Can be shared between different Gradle daemons |
+| In process     | Inside the Gradle daemon process  | No                                  | May share heap with the Gradle daemon                                  |
+| Out of process | In a separate forked process      | No                                  | -                                                                      |
```

Comment:
Not sure if `forked` adds meaning here. Does it have any practical use?

Reply:
For me, it only raised questions "Forked from what? What should I do with the fact that it was forked?"

Reply:
@ALikhachev could you help, please?

Reply:
Yep, I guess we can remove `forked`. It's a technical detail
I'd suggest to rephrase it to be more clear
like
```suggestion
| Out of process | In a separate process for each call      | No                                  | -                                                                      |
```
So the new description emphasises that a new process will be created for each compilation task.